lib/active_record_host_pool.rb

```diff
@@ -15,7 +15,11 @@ class << self
   ActiveRecordHostPool.loaded_db_adapter = :mysql2
 elsif Gem.loaded_specs.include?("trilogy")
   require "trilogy"
-  require "activerecord-trilogy-adapter" if ActiveRecord.version < Gem::Version.new("7.1")
+  if ActiveRecord.version < Gem::Version.new("7.1")
+    require "activerecord-trilogy-adapter"
+    require "trilogy_adapter/version"
+    raise "ActiveRecordHostPool is only compatible with activerecord-trilogy-adapter v3.1+" if Gem::Version.new("3.1") > TrilogyAdapter::VERSION
```

Comment:
```suggestion
    raise "ActiveRecordHostPool is only compatible with activerecord-trilogy-adapter v3.1+" if TrilogyAdapter::VERSION < Gem::Version.new("3.1")
```

Reply:
Standardrb made me switch this.

Reply:
What, standardrb says line 18 and 21 are both fine, even if they check in opposite order? It’s the YodaCondition cop, right?

Reply:
It is:

```
  lib/active_record_host_pool.rb:21:96: Style/YodaCondition: Reverse the order of the operands `TrilogyAdapter::VERSION < Gem::Version.new("3.1")`.
```